paydb_endpoint.py

```diff
@@ -252,7 +262,27 @@ def user_info():
         return jsonify(dict(email=user.email, balance=balance, photo=user.photo, photo_type=user.photo_type, roles=roles, permissions=perms))
     return jsonify(dict(email=user.email, balance=-1, photo=user.photo, photo_type=user.photo_type, roles=[], permissions=[]))
 
+@paydb.route('/user_reset_password', methods=['GET', 'POST'])
+@limiter.limit("3/hour")
+def user_reset_password():
+    sig = request_get_signature()
+    content = request.get_json(force=True)
+    if content is None:
+        return bad_request(web_utils.INVALID_JSON)
+    params, err_response = get_json_params(content, ["api_key", "nonce"])
+    if err_response:
+        return err_response
+    api_key, nonce = params
+    res, reason, api_key = check_auth(db.session, api_key, nonce, sig, request.data)
+    if not res:
+        return bad_request(reason)
+    user = api_key.user
+    #### separate DB for the instructions(flask-security)
```

Comment:
remove this comment, the same DB is used, it is just a different module that handles the user/login etc

Reply:
removed